test/common/mock_admin_data.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Let's have a different folder under `test` and move all the files related to each of these APIs there
```
admin/
spender/
approver/
common/
```